schema.graphql

```diff
@@ -5,10 +5,15 @@ type Domain @entity {
   name: String
   labelHash: String
   parent: Domain
-  subdomains: [Domain!]
+  subdomains: [Domain!] @derivedFrom(field: "parent")
   owner: Account
   creator: Account
   transactionId: Bytes
+  lockedBy: Account
+  isLocked: Boolean
+  metaData: String
+  royalty: Int
```

Comment:
Not type safe with ethereum, use BigInt instead

Reply:
This I was able to implement